ProcessLib/ThermoMechanicalPhaseField/ThermoMechanicalPhaseFieldFEM-impl.h

```diff
@@ -0,0 +1,644 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ *  \file   ThermoMechanicalPhaseFieldFEM-impl.h
+ *  Created on January 8, 2018, 3:00 PM
+ */
+#pragma once
+
+#include "ThermoMechanicalPhaseFieldFEM.h"
+
+#include "NumLib/DOF/DOFTableUtil.h"
+#include "ProcessLib/CoupledSolutionsForStaggeredScheme.h"
+
+namespace ProcessLib
+{
+namespace ThermoMechanicalPhaseField
+{
+template <typename ShapeFunction, typename IntegrationMethod,
+          int DisplacementDim>
+void ThermoMechanicalPhaseFieldLocalAssembler<ShapeFunction, IntegrationMethod,
+                                DisplacementDim>::
+    assembleWithJacobian(double const t, std::vector<double> const& local_x,
```

Comment:
This function is not tested with the ctest, but should be.

Reply:
the monolithic scheme is removed in current process.